File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 2 - C/q5.c
//Crie um algoritmo que leia um número inteiro do usuário e imprima uma mensagem se
//o número digitado for par ou ímpar. Para verificar se um número é par, utilizamos o operador %
//(resto da divisão). Ao dividir um número por 2 e o resto da divisão for igual a zero, temos um
//número par.
 
#include<stdio.h>
int main(){
	int x;
	printf("Insira um valor: ");
	scanf("%d", &x);
	if(x%2==0){
		printf("\nPar");
	}
	else{
		printf("\nImpar");
	}
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 2 - Python/q5.py
#Crie um algoritmo que leia um número inteiro do usuário e imprima uma mensagem se
#o número digitado for par ou ímpar. Para verificar se um número é par, utilizamos o operador %
#(resto da divisão). Ao dividir um número por 2 e o resto da divisão for igual a zero, temos um
#número par.

n=int(input("digite o valor de n: "))
resto=(n%2)
if resto==0:
    print("e par")    
else:
    print("e impar ")

     
